nf_core/modules/remove.py

```diff
@@ -58,5 +58,27 @@ def remove(self, module):
 
         log.info("Removing {}".format(module))
 
+        # Remove entry from modules.json
+        if not self.remove_modules_json_entry(module):
+            return False
+
         # Remove the module
         return self.clear_module_dir(module_name=module, module_dir=module_dir)
+
+    def remove_modules_json_entry(self, module):
+        # Load 'modules.json'
+        modules_json_path = os.path.join(self.dir, "modules.json")
+        with open(modules_json_path, "r") as fh:
+            modules_json = json.load(fh)
+        if module in modules_json.get("modules", {}):
+            modules_json["modules"].pop(module)
+        else:
+            # If the module is missing from modules.json we can still clear the directory
+            # in which the untracked module resides
+            log.error(f"Module '{module}' is missing from 'modules.json' file.")
+            return questionary.confirm(
+                f"Do you wish to continue removing module '{module}'?", default=False
+            ).unsafe_ask()
```

Comment:
I would say we don't need a prompt here, maybe just say that a `modules.json` is missing.

Reply:
Yeah, I guess we can rely on the user knowing `rm -rf` atleast.